tests/unit/ops/transformer/inference/test_quant.py

```diff
@@ -0,0 +1,38 @@
+import torch
+import pytest
+from deepspeed.ops import op_builder
+
+quantizer_cuda_module = op_builder.QuantizerBuilder().load()
```

Comment:
I believe this is causing the unrelated tests to fail. Let's refactor this to use the loading style used either [here](https://github.com/microsoft/DeepSpeed/blob/a691ec605b2383ecac21ecf33a9ba1c0cba4dd01/tests/unit/ops/transformer/inference/test_bias_gelu.py#L42).

Reply:
@GuanhuaWang -- yes, please see the code above. We set the module to none at global level and only load it inside the function that needs it.

Reply:
@cmikeh2 @awan-10  move the test file to `tests/unit/ops/quantizer`, also used the loading style used either [here](https://github.com/microsoft/DeepSpeed/blob/a691ec605b2383ecac21ecf33a9ba1c0cba4dd01/tests/unit/ops/transformer/inference/test_bias_gelu.py#L42).